web-app/.env.dev.dist

```
@@ -2,4 +2,5 @@ TYPEORM_CONNECTION=postgres
 TYPEORM_HOST=postgres
 TYPEORM_USERNAME=postgres
 TYPEORM_PASSWORD=postgres
-TYPEORM_DATABASE=test
\ No newline at end of file
+TYPEORM_DATABASE=test
+JWT_SECRET=aaabbbccc
```

```suggestion
JWT_SECRET=!ChangeMe!
```

We generally use this as default value to not forget to change it :)